docs/cog_guides/admin.rst

```diff
@@ -0,0 +1,427 @@
+
+.. _admin-command-addrole:
+
+^^^^^^^
+addrole
+^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]addrole <rolename> [user]
+
+**Description**
+
+Adds a role to a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to add to the member. |role-input|
+
+* ``[user=ctx]``: The member you want to add the role to. Defaults to the
+  command author. |member-input|
+
+.. _admin-command-removerole:
+
+^^^^^^^^^^
+removerole
+^^^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]removerole
+
+**Description**
+
+Removes a role from a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to remove. |role-input|
+
+* ``[user=ctx]``: The member to remove the role from. |member-input| Defaults to
+  the command author.
+
+.. _admin-command-editrole:
+
+^^^^^^^^
+editrole
+^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole
+
+**Description**
+
+Edits the settings of a role.
+
+.. _admin-command-editrole-name:
+
+"""""""""""""
+editrole name
+"""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole name <role> <name>
+
+**Description**
+
+Edits the name of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input|
+
+* ``<name>``: The new role name.
+
+.. _admin-command-editrole-color:
+
+""""""""""""""
+editrole color
+""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole color <role> <color>
+
+**Description**
+
+Edits the color of a role.
+
+**Arguments**
+
+* ``<role>``: The role name to edit. |role-input|
+
+* ``<color>``: The new color to assign. |color-input|
+
+**Examples**
+
+* ``[p]editrole color "My role" #ff0000``
+
+* ``[p]editrole color "My role" dark_blue``
+
+.. _admin-command-announce:
+
+^^^^^^^^
+announce
+^^^^^^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announce <message>
+
+**Description**
+
+Announce your message to all of the servers the bot is in.
+
+The bot will announce the message in the guild's announcements channel
+if set, else she will try the system channel (where the new members are
+welcomed with the Discord announcer). If none of these channels are found,
+the bot will use the highest channel in the list.
+
+**Arguments**
+
+* ``<message>``: The message to send.
+
+.. _admin-command-announce-cancel:
+
+"""""""""""""""
+announce cancel
+"""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announce cancel
+
+**Description**
+
+Cancels an active announcement.
+
+.. _admin-command-announceset:
+
+^^^^^^^^^^^
+announceset
+^^^^^^^^^^^
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]announceset
+
+**Description**
+
+Manage the settings of announcements sent by the bot owners on servers.
```

Comment:
I'm not sure this makes sense and I don't know how to make it make sense.

Reply:
`Change how announcements are sent in this guild.` is the command's docstring.